### ISSUE TITLE
Https connections on marshmallow

### ISSUE THREAD

**Issue by SashaKz** [ogrebgr/khandroid#2]
Hi, Ognyan!

Will khandroid work on marshmallow?
I changed all http connection's classes for khandroid, and it's now successfully compiled for:
buildToolsVersion '23.0.1', but I didn't run it on marshmallow. It seems you are using apache libraries, may be you know, will it work on marshmallow?

**Comment by ogrebgr** [comment]
Yes, it should work on marshmallow without any problems. The apache library HttpClient is pure java, i.e. it is not android dependant so it should work (I did not tested it yet).

**Comment by SashaKz** [comment]
Thanks a lot for your consolatory answer, my friend
